ios/shpeucfapp/Info.plist

```diff
@@ -22,6 +22,13 @@
 	<string>????</string>
```

Comment:
I made a comment above inside of the first podfile. I don't think git showed it to you but i'll just copy and paste it here,
@grindlock
Hey I guess i forgot to submit my pr from yesterday.
Please remove all pod file changes from this pr.
Try looking at the commands available here. https://ohshitgit.com/
As a tip whenever you get rid of commits that were already pushed you need to do a force push

Reply:
I resolve this. did a rebase to solve it.